src/idlcxx/src/streamers.c

```diff
@@ -253,8 +254,23 @@ check_endpoint(
     fmt = "  %4$s::%2$s(__str, %1$s);\n";
   }
 
+  /**
```

Comment:
Use a single `*` please. `/**` indicates api documentation to Doxygen, normal comments should use `/*`.

Reply:
This is no longer in the code.

Reply:
This code is now outdated.